charts/eth-validator-watcher/templates/service.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: v1
```

Comment:
We can keep service if external users want to expose the eth-watcher (e.g an internal LB) even if currently the watcher does not support multiple replicas

and we have a section dedicated to service in the values.yaml